app/ui/cypress/integration/projectNavigationBar.spec.ts

```diff
@@ -0,0 +1,73 @@
+describe('Project Navigation Bar Behavior', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3001');
+    cy.getByTestId('project').first().parent().click();
+  });
+
+  it('should show the overview project page', () => {
+    // Assert.
+    cy.url().should('include', '/projects/').and('include', 'overview');
```

Comment:
Would this be better?
```suggestion
    cy.url().should('include', '/overview');
```